
enum DbOperateType 
{
    MSG_DB_WORK_START = 10000;//common update
    MSG_DB_GET_USER_INFO = 10001;//Get user info
}



message DBUserInfo
{
    required int32 user_id = 1;
    required string user_name = 2;
}

message DBCommonReq
{
    required int32  operate_type = 1;
    required string operate_string = 2;
}

message DBCommonRep
{
    required int32 ret = 1;
    required int32 operate_type = 2;
    optional DBUserInfo user_info = 3;
}


